frontend/src/citizen-frontend/messages/ThreadListItem.tsx

```diff
@@ -59,15 +63,28 @@ export default React.memo(function ThreadListItem({
         <Truncated>
           {lastMessage.content.substring(0, 200).replace('\n', ' ')}
         </Truncated>
+        {lastMessage.attachments.length > 0 && (
+          <FixedSpaceColumn spacing="xs">
+            {lastMessage.attachments.map((attachment) => (
+              <FileDownloadButton
+                key={attachment.id}
+                file={attachment}
+                fileFetchFn={getAttachmentBlob}
+                onFileUnavailable={onAttachmentUnavailable}
+                icon
+                data-qa="thread-list-attachment"
+              />
+            ))}
+          </FixedSpaceColumn>
+        )}
       </FixedSpaceColumn>
     </Container>
   )
 })
 
-const Container = styled.button<{ isRead: boolean; active: boolean }>`
+const Container = styled.div<{ isRead: boolean; active: boolean }>`
```

Comment:
does this need e.g. `cursor: pointer;`

Reply:
It's there on row 91 🙂
Button was invalid markup, because of the new file download button: no buttons within buttons.